RetakesPlugin/Modules/Managers/GameManager.cs

```diff
@@ -66,40 +69,34 @@ public void AddScore(CCSPlayerController player, int score)
 
     private int _consecutiveRoundsWon;
 
-    public void TerroristRoundWin()
+    private void TerroristRoundWin()
     {
         _consecutiveRoundsWon++;
 
-        if (_consecutiveRoundsWon == _consecutiveRoundWinsToScramble)
+        bool shouldScrambleNow = _isScrambleEnabled && _consecutiveRoundsWon == _consecutiveRoundWinsToScramble;
```

Comment:
As the types of these variables can be inferred by the declaration, use var instead.

Reply:
Sure, for bool I think it is kind of nice to see the type. But I will change it to fix your style.